src/routes/(console)/project-[region]-[project]/functions/function-[function]/domains/add-domain/verify-[domain]/+page.svelte

```diff
@@ -28,24 +28,24 @@
     let { data } = $props();
 
     const ruleId = page.url.searchParams.get('rule');
-    const domainId = page.url.searchParams.get('domain');
     const isSubDomain = $derived.by(() => isASubdomain(page.params.domain));
 
-    let selectedTab = $state<'cname' | 'nameserver' | 'a' | 'aaaa'>('nameserver');
-    $effect(() => {
-        if ($regionalConsoleVariables._APP_DOMAIN_TARGET_CNAME && isSubDomain) {
-            selectedTab = 'cname';
-        } else if (!isCloud && $regionalConsoleVariables._APP_DOMAIN_TARGET_A) {
-            selectedTab = 'a';
-        } else if (!isCloud && $regionalConsoleVariables._APP_DOMAIN_TARGET_AAAA) {
-            selectedTab = 'aaaa';
-        } else {
-            selectedTab = 'nameserver';
-        }
-    });
-    let verified: boolean | undefined = $state(undefined);
+    let selectedTab = $state<'cname' | 'nameserver' | 'a' | 'aaaa'>(
+        (() => {
+            if ($regionalConsoleVariables._APP_DOMAIN_FUNCTIONS && isSubDomain) {
+                return 'cname';
+            } else if (!isCloud && $regionalConsoleVariables._APP_DOMAIN_TARGET_A) {
+                return 'a';
+            } else if (!isCloud && $regionalConsoleVariables._APP_DOMAIN_TARGET_AAAA) {
+                return 'aaaa';
+            } else {
+                return 'nameserver';
+            }
+        })()
+    );
```

Comment:
err, lets make a separate function to get the initial value. we avoid IIFEs like these.